include/net/net_pkt.h

```diff
@@ -240,6 +240,12 @@ struct net_pkt {
 	uint8_t ieee802154_arb : 1; /* ACK Request Bit is set in the frame */
 	uint8_t ieee802154_ack_fpb : 1; /* Frame Pending Bit was set in the ACK */
 	uint8_t ieee802154_frame_retry : 1; /* The frame is being retransmitted. */
+	uint8_t ieee802154_frame_secured : 1; /* Frame is authenticated and
+					       * encrypted according to its
+					       * Auxiliary Security Header
+					       */
+	uint8_t ieee802154_fc_is_set : 1; /* Frame Counter was set */
+	uint8_t ieee802154_ie_is_updated : 1; /* Information Elements were updated */
```

Comment:
I would extend this comment a little to indicate that only selected IEs can be updated by radio. The list includes IEs with precise timing data unknown by the upper layer. I think the only one so far is CSL IE.

Reply:
Please check if the current doc is accurate enough.

Reply:
Perfect!